core/packages/PackageDB.h

```diff
@@ -49,12 +49,17 @@ class PackageDB final {
 
     const std::string_view errorHint() const;
 
+    const std::string_view getLayerName(uint8_t layerRank) const;
+    const uint8_t getLayerRank(std::string layerName) const;
```

Comment:
Could this take a `const &string` or a string_view?  Doesn't apply if the API ends up changing.